### PR TITLE
Feacture 09 02 2023 ab account page

### PR DESCRIPTION
**Pull request by Balaji-agidigalla**
Pull Request (PR) Request Template:

### Problem
<Provide a brief overview of the problem or task that this PR addresses.>
1. Profile page setup - Current page looks very basic and scattered, we need align everything properly

Monday Card Link: [<https://marmeto-force.monday.com/boards/5966670929/pulses/6025395742>]

### Solution
<Briefly describe what was implemented and how it solves the problem.>
i have refered the other websites account page and i just make changes on current theme

<Visual representation or screenshots>
image

### Impacted Areas
<Explain other areas of the application that could be affected by this change.>
Account Page 

### Done Checklist
- [x] Performed unit testing to ensure functionality and identify potential issues.
- [x] Followed basic code standards and best practices. (indentation, media queries, basic customizer things)
- [x] Added comments to explain complex logic or improve code readability.
- [x] Followed Naming conventions and proper variable declaration.
- [x] Handled edge cases and errors gracefully to prevent crashes.
- [x] Optimized the code for production to ensure performance.

Please review this PR and provide feedback. Thank you!

### CODE REVIEW

File: assets/account-page-tabs.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+function openTab(evt, tabId) {
+  var i, tabContents, tabLinks;
+  tabContents = document.querySelectorAll(".tab-content");
+  for (i = 0; i < tabContents.length; i++) {
```
Comment (arshiya-khanam) [inline assets/account-page-tabs.js:4]: you can use forEach than doing forloop

File: assets/account-page-tabs.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    tabContents[i].style.display = "none";
+  }
+  tabLinks = document.querySelectorAll(".tablinks");
+  for (i = 0; i < tabLinks.length; i++) {
```
Comment (arshiya-khanam) [inline assets/account-page-tabs.js:8]: you can use forEach than doing forloop

File: assets/account-page-tabs.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+document.addEventListener("DOMContentLoaded", function() {
+  var activeElement = document.querySelector(".is--active");
+  if (activeElement) {
+    activeElement.click();
+  }
+  
+  var tabLinks = document.querySelectorAll('.tablinks');
+  for (var i = 0; i < tabLinks.length; i++) {
```
Comment (arshiya-khanam) [inline assets/account-page-tabs.js:28]: you can use forEach than doing forloop

File: templates/customers/order.liquid [hunk 4]
```diff
@@ -32,7 +32,7 @@
         <thead>
           <tr>
             <th>{{ 'customer.order.product' | t }}</th>
-            <th>{{ 'customer.order.sku' | t }}</th>
+            {% comment %} <th>{{ 'customer.order.sku' | t }}</th> {% endcomment %}
```
Comment (arshiya-khanam) [inline templates/customers/order.liquid:35]: remove unwanted code

File: templates/customers/order.liquid [hunk 5]
```diff
@@ -81,7 +81,7 @@
                 </div>
               {% endif %}
             </td>
-            <td data-label="{{ 'customer.order.sku' | t }}">{{ line_item.sku }}</td>
+            {% comment %} <td data-label="{{ 'customer.order.sku' | t }}">{{ line_item.sku }}</td> {% endcomment %}
```
Comment (arshiya-khanam) [inline templates/customers/order.liquid:84]: remove unwanted code

**Review by arshiya-khanam — CHANGES_REQUESTED** [review]

**Review by arshiya-khanam — APPROVED** [review]

**Review by Manuraj-Marmeto — APPROVED** [review]

**Pull request merged**